<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyGameLauncherClass</class>
 <widget class="QMainWindow" name="MyGameLauncherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyGameLauncher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titlebar_widget_" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButton_hide">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MyGameLauncher.qrc">
           <normaloff>:/MyGameLauncher/resource/tob_none_btn01.png</normaloff>:/MyGameLauncher/resource/tob_none_btn01.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_resize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MyGameLauncher.qrc">
           <normaloff>:/MyGameLauncher/resource/tob_none_btn02.png</normaloff>:/MyGameLauncher/resource/tob_none_btn02.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_exit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="MyGameLauncher.qrc">
           <normaloff>:/MyGameLauncher/resource/exit.png</normaloff>:/MyGameLauncher/resource/exit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="main_application_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,7,2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="game_list_widget_" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="main_widget_" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="8,2">
          <item>
           <widget class="QLabel" name="label_image_">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>36</pointsize>
             </font>
            </property>
            <property name="text">
             <string>좌 측 게임선택 버튼을 선택하세요</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,1,1">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_path_">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>PATH: c:/mylauncher/PCGAME</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_game_title_">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>GAME</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_down_">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {border: 1px solid black; background-color:#4287f5; color: #021533;}

QPushButton:pressed {border: 1px solid black; background-color:#364a69; color: white;}

QPushButton:hover {border: 1px solid black; background-color:#353a42; color: white;}</string>
                 </property>
                 <property name="text">
                  <string>DOWN</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_delete_">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {border: 1px solid black; background-color:#4287f5; color: #021533;}

QPushButton:pressed {border: 1px solid black; background-color:#364a69; color: white;}

QPushButton:hover {border: 1px solid black; background-color:#353a42; color: white;}</string>
                 </property>
                 <property name="text">
                  <string>DELETE</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_dir_">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>16</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {border: 1px solid black; background-color:#4287f5; color: #021533;}

QPushButton:pressed {border: 1px solid black; background-color:#364a69; color: white;}

QPushButton:hover {border: 1px solid black; background-color:#353a42; color: white;}</string>
                 </property>
                 <property name="text">
                  <string>SET DIR</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="login_status_widget_" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_id_">
              <property name="text">
               <string>ID : ACCOUNT</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_vip_">
              <property name="text">
               <string>VIP :</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QScrollArea" name="scrollArea_friendlist_">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="friend_list_widget_">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>179</width>
                 <height>534</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyGameLauncher.qrc"/>
 </resources>
 <connections/>
</ui>
